Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 find_package(Qt5Widgets REQUIRED)
-find_Package(Qt5Gui REQUIRED)
+find_package(Qt5Gui REQUIRED)
+
+if(NOT Qt5Widgets_FOUND)
+  message(FATAL_ERROR "Could not find Qt5, which is required to build the DolphinQt2 frontend. If you don't want to build the Qt frontend, pass -DENABLE_QT2=False to
```

Comment:
Would this ever get hit? I thought `find_package(<package name> REQUIRED)` always halts processing with an error if the package couldn't be found.

Reply:
Oh, yeppp.

Reply:
Removed ``REQUIRED`` from both ``Qt5Gui`` and ``Qt5Widgets``